protos/telemetry/telemetry.proto

```diff
@@ -118,6 +136,12 @@ message EulerAngle {
     float yaw_deg = 3;
 }
 
+message AngularSpeed {
+    float rollspeed = 1;
+    float pitchspeed = 2;
+    float yawspeed = 3;
+}
```

Comment:
I'm confused here. What is this for? Is it for this...

https://github.com/mavlink/MAVSDK/blob/935988bbf398cb4eb7c274b698e137073a8c9434/src/plugins/telemetry/include/plugins/telemetry/telemetry.h#L134-L143

Reply:
This is for current angular speed from ATTITUDE_QUATERNION message:
https://github.com/mavlink/MAVSDK/pull/832/files#diff-5d3c38fd06bf641a3131f56ff3b57f05R117

Reply:
Hm, and that's not the same as `AngularVelocityNED` I suppose but rather in body frame? Could we get the naming closer to `AngularVelocityNED`?

Reply:
Yes, this is from [this msg](https://mavlink.io/en/messages/common.html#ATTITUDE_QUATERNION), in body frame.

I think that naming really could be closer.
How about 
```
message AngularVelocityBF {
    float roll_rad_s = 1;
    float pitch_rad_s = 2;
    float yaw_rad_s = 3;
}
```

Reply:
```
message AngularVelocityBody {
    float roll_rad_s = 1;
    float pitch_rad_s = 2;
    float yaw_rad_s = 3;
}
```

There are others called `...Body` so that would be consistent.

Reply:
Ok. Will do it